src/ray/raylet/dependency_manager.h

```diff
@@ -192,11 +192,8 @@ class DependencyManager : public TaskDependencyManagerInterface {
 
   /// A struct to represent the object dependencies of a task.
   struct TaskDependencies {
-    TaskDependencies(const std::vector<rpc::ObjectReference> &deps)
-        : num_missing_dependencies(deps.size()) {
-      const auto dep_ids = ObjectRefsToIds(deps);
-      dependencies.insert(dep_ids.begin(), dep_ids.end());
-    }
+    TaskDependencies(absl::flat_hash_set<ObjectID> &&deps)
```

Comment:
R-value ref along with the move above is a bit suspicious, may explain the windows segfault.